neuro_extras/main.py

```diff
@@ -482,12 +505,12 @@ def data_cp(
     destination: str,
     extract: bool,
     compress: bool,
-    volume: Sequence[str],
-    env: Sequence[str],
+    volume: List[str],
+    env: List[str],
 ) -> None:
     if extract and compress:
         raise click.ClickException("Extract and compress can't be used together")
-    run_async(_data_cp(source, destination, extract, compress, volume, env))
+    run_async(_data_cp(source, destination, extract, compress, list(volume), list(env)))
```

Comment:
already list? (see typing in method signature)

Reply:
Click gives us a non-mutable sequence, so we have to convert this to a list. I'll update the signature to reflect correct type.